### ISSUE TITLE
Could not create DMG archive

### ISSUE THREAD

**Issue by mazura1972** [aonez/Keka#1315]
### Configuration
- Keka version: 1.3.3
- macOS version: 13.5.1

### Describe the bug
No DMG archive is created when I drag a file into KEKA application or use Finder -> Compress using Keka local menu. If in the configuration I choose another default compression format, e.g. 7z, zip and others, an appropriate archive is created.
I'm still using the old version 1.0.12 of Keka and it works fine there.

### To Reproduce
1. Drag and drop any file into Keka window
2. You can see that nothing happens. The compression progress window does not appear. 

### Expected behavior
In the source location of the compressed file, a dmg file should appear.

**Comment by aonez** [comment]
Due to sandbox limitations Keka can only create DMG's from a folder since version 1.1.0. You can workaround this by just temporarily moving the files you want to a folder and then drop the folder to Keka.

Probably you have dismissed the warning stating this:

<img width="372" alt="Screenshot 2023-08-24 at 22 12 47" src="https://github.com/aonez/Keka/assets/480163/944ee5a8-02ca-44a4-abd6-0d4c1a7d5245">

You can enable that message again by executing this command in the Terminal.app and restarting Keka:
```sh
defaults remove com.aone.keka DMGWarning
```

**Comment by mazura1972** [comment]
Thank you for the quick reply. Running the defaults remove command helped display this message.
With the help of Keka, I make a compressed archive of my MacOS applications, i.e. I try to compress a folder called SampleProgram.app. Are you suggesting that I should additionally put this folder in another folder and then compress it like this, do I understand correctly?

**Comment by aonez** [comment]
That's right. The .app is considered a bundle, not a folder, even if it is indeed a folder. Just move them to a folder a drag that folder instead.

**Comment by mazura1972** [comment]
It actually works this way. It just doesn't give me peace of mind why the compression of this type of directories (e.g. Siri.app) works for other formats like 7z and the rest of them.
Thanks for your help, best regards.
